contracts/GnosisSafe.sol

```diff
@@ -62,330 +16,7 @@ contract GnosisSafe {
-            require(newContract != 0);
-            ContractCreation(newContract);
-        }
-    }
-
-    function executeCall(address to, uint256 value, bytes data)
-        internal
-        returns (bool success)
-    {
-        assembly {
-            success := call(not(0), to, value, add(data, 0x20), mload(data), 0, 0)
-        }
-    }
-
-    function executeDelegateCall(address to, bytes data)
-        internal
-        returns (bool success)
-    {
-        assembly {
-            success := delegatecall(not(0), to, add(data, 0x20), mload(data), 0, 0)
-        }
-    }
-
-    function executeCreate(bytes data)
-        internal
-        returns (address newContract)
-    {
-        assembly {
-            newContract := create(0, add(data, 0x20), mload(data))
-        }
-    }
-
-    /// @dev Returns transactions hash to be signed by owners.
-    /// @param to Destination address.
-    /// @param value Ether value.
-    /// @param data Data payload.
-    /// @param operation Operation type.
-    /// @param _nonce Transaction nonce.
-    /// @return Transaction hash.
-    function getTransactionHash(address to, uint256 value, bytes data, Operation operation, uint256 _nonce)
-        public
-        view
-        returns (bytes32)
-    {
-        return keccak256(byte(0x19), this, to, value, data, operation, _nonce);
-    }
-
-    /// @dev Returns array of owners.
-    /// @return Array of Safe owners.
-    function getOwners()
-        public
-        view
-        returns (address[])
-    {
-        return owners;
-    }
-
-    /// @dev Returns array of extensions.
-    /// @return Array of extensions.
-    function getExtensions()
-        public
-        view
-        returns (Extension[])
-    {
-        return extensions;
-    }
-
-    /// @dev Returns a the count of owners that have confirmed the given transaction.
-    /// @param transactionHash Safe transaction hash.
-    function getConfirmationCount(bytes32 transactionHash)
-        public
-        view
-        returns (uint confirmationCount)
-    {
-        for (uint i = 0; i < owners.length; i++) {
-            if (isConfirmed[owners[i]][transactionHash])
-                confirmationCount++;
-        }
-    }
-
-    /// @dev Returns a list of owners that have confirmed the given transaction.
-    /// @param transactionHash Safe transaction hash.
-    function getConfirmingOwners(bytes32 transactionHash)
-        public
-        view
-        returns (address[] confirmingOwners)
-    {
-        uint confirmationCount = getConfirmationCount(transactionHash);
-        confirmingOwners = new address[](confirmationCount);
-        confirmationCount = 0;
-        for (uint i = 0; i < owners.length; i++) {
-            if (isConfirmed[owners[i]][transactionHash]) {
-                confirmingOwners[confirmationCount] = owners[i];
-                confirmationCount++;
-            }
-        }
+        setupOwners(_owners, _threshold);
```

Comment:
Constructors of inherited contracts can be called like modifiers (https://github.com/gnosis/MultiSigWallet/blob/master/contracts/MultiSigWalletWithDailyLimit.sol#L30). I guess this won't work for regular functions (not sure)?

Reply:
not as far as I understand the docs.